packages/front-end/services/dates.ts

```diff
@@ -36,7 +36,18 @@ export function getValidDate(
 
   if (!dateStr) return fallback;
 
-  const d = new Date(dateStr);
+  let d: Date;
+
+  if (typeof dateStr === "string") {
+    // Fix date parsing for Safari
+    let reformattedDateStr = dateStr.replace(/[/\s]/g, "-");
+    if (reformattedDateStr.match(/-/g)?.length === 1) {
+      reformattedDateStr = reformattedDateStr.replace("-", "-1-");
```

Comment:
Does the date-fns library we're using have any utilities for parsing the date? I'd think that might be safer.

Reply:
Sort of, there is `parse`, but it doesn't reliably work in this use case. Would rather patch this quickly and investigate further post launch.

Reply:
I have a feeling this patch might end up causing more bugs than it fixes. Do you have an example string that is being parsed incorrectly in Safari and how this regex fixes it?

I'd also like to know why the date string is invalid in the first place. This might be something we can just quickly patch on the back end for example.